README.md

```diff
@@ -1,282 +1,584 @@
+
 # Security Hub Visibility
 
+  
+
 ## A GitHub Action that syncs Security Hub findings with Jira issues
 
+  
+
 1. Gets all current findings from Security Hub, filtered by the `Severity` of the findings (default is `CRITICAL,HIGH,MEDIUM`)
 
+  
+
 2. Creates Jira issues for findings that do not already have a Jira issue
 
-- **To avoid creating duplicate issues**, the search criteria use the custom label configuration as well as default configuration for three identifying search labels: region, AWS account ID, and "security-hub."
+  
+
+-  **To avoid duplicate issues**, the search criteria use default settings for three primary identifiers: region, AWS account ID, and "security-hub," along with any custom label configurations provided (Note that modifications or deletions of labels may lead to duplicate issue creation). Additionally, a comparison of the Jira issue description and finding title is made against the findings list. If there is a title match, the decision is based on the Resources list. 
```

Comment:
Would it make sense to rename "security-hub," to "Security Hub title" or something a bit more descriptive?

Reply:
Can you provide an example of "custom label configuration" here? 

It's good that you included the "Note" about modifying custom labels here. Can this be bolded/italicized and on a different line to make sure it's clearly visible?

Reply:
> **make sense to rename "security-hub," to "Security Hub titl**
So in this paragraph we are referring to the Label to identify every issue that is created by this automation.

Reply:
> > **make sense to rename "security-hub," to "Security Hub titl**
> > So in this paragraph we are referring to the Label to identify every issue that is created by this automation.

I understand you point - ill update the readme to specify "labels" as the identifiers

Reply:
@yonassrobi 

> **Can you provide an example of "custom label configuration" here?**
Example: `jira-labels-config:` `"[{\"labelField\":\"ProductName\",\"labelPrefix\":\"product\",\"labelDelimiter\":\":\"},{\"labelField\":\"severity\"},{\"labelField\":\"accountId\",\"labelDelimiter\":\":\",\"labelPrefix\":\"account\"},{\"labelField\":\"region\"}]"`

Result of the custom labeling above the labels on the issues created will be: "product:Tenable" "High" "account:abc123" "us-east-1"

_Ill add an example on the readme too_

> **It's good that you included the "Note" about modifying custom labels here. Can this be bolded/italicized and on a different line to make sure it's clearly visible?**

Yes Ill do that.